sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_registry_client.py

```diff
@@ -442,7 +442,7 @@ def list_tag_properties(self, repository: str, **kwargs: Any) -> AsyncItemPaged[
 
         :param str repository: Name of the repository
         :keyword order_by: Query parameter for ordering by time ascending or descending
-        :paramtype order_by: :class:`~azure.containerregistry.TagOrder` or str
+        :paramtype order_by: :class:`~azure.containerregistry.ArtifactTagOrder` or str
```

Comment:
We shouldn't use the `:class:` tag within `:paramtype:`  - I think it should just be:
`:paramtype order_by: ~azure.containerregistry.ArtifactTagOrder or str`

Reply:
Updated.

Reply:
Please also remove the angle quotes - they shouldn't be around the object, just:
~azure.containerregistry.ArtifactTagOrder